source/gui/message.py

```diff
@@ -1,16 +1,30 @@
 # -*- coding: UTF-8 -*-
```

Comment:
extra notes:

- isModalMessageBoxActive should be deprecated and the logic moved to `BlockingInstancesExist`. The purposes of these functions are the same and we should avoid duplications. I would suggest deprecating `displayDialogAsModal` too in favour of `MessageDialog.showModal`
-

Reply:
Fair enough, thanks for the explanation